examples/examples-cpu/snowflake/snowflake-dask.ipynb

```diff
@@ -0,0 +1,388 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snowflake + Dask\n",
+    "\n",
+    "How to load data from a Snowflake table or query into a Dask dataframe"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to Snowflake\n",
+    "\n",
+    "See [README](README.md) for more details on how to set up the credentials file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import yaml\n",
+    "import snowflake.connector\n",
```

Comment:
does the image or start script need to change? I'd be surprised to learn that `snowflake-connector-python` is in our default images, since it's such a heavy dependency

Reply:
`snowflake-connector-python` _is_ in our default image: [here](https://github.com/saturncloud/images/blob/5afc87a15da762c82c6a7c78ade38b9cf7a04aa8/saturn/environment.yml#L44). But I think there are missing dependencies in general. For instance none of the dashboard dependencies are in the default image or installed in the config file for this project.

Reply:
created tasks to update images will do today

Reply:
https://github.com/saturncloud/images/pull/65